changelog/fix_fix_empty_lines_around_method_body_for_methods.md

```diff
@@ -0,0 +1 @@
+* [#13430](https://github.com/rubocop/rubocop/pull/13430): Fix EmptyLinesAroundMethodBody for methods with arguments spanning multiple lines. ([@aduth][])
```

Comment:
```suggestion
* [#13430](https://github.com/rubocop/rubocop/issues/13430): Fix EmptyLinesAroundMethodBody for methods with arguments spanning multiple lines. ([@aduth][])
```

Reply:
Whoops, I was trying to be clever about getting the "next" PR number in here ahead of actually creating the pull request, but my issue took the number I was expecting for the PR.

Would you prefer that this point to the issue #13430 ?

```suggestion
* [#13430](https://github.com/rubocop/rubocop/issues/13430): Fix EmptyLinesAroundMethodBody for methods with arguments spanning multiple lines. ([@aduth][])
```

...or the pull request #13431 ?

```suggestion
* [#13431](https://github.com/rubocop/rubocop/pull/13431): Fix EmptyLinesAroundMethodBody for methods with arguments spanning multiple lines. ([@aduth][])
```

Reply:
When there's an issue, we link the changelog to the issue, not the PR. Only if there is only a PR without an issue do we link to the PR.

Reply:
> When there's an issue, we link the changelog to the issue, not the PR. Only if there is only a PR without an issue do we link to the PR.

Makes sense 👍 Updated in e5b58d0.